config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also add a default route(i.e. `root` route) as right now it shows the rails logo. To help you doing that just update the line 5 to something like this: `root "users#index"` and uncomment it.